### ISSUE TITLE
Provide an intro to unconferencing

### ISSUE THREAD

**Issue by vmbrasseur** [dcppc/dcppc-workshops#8]
Some guidance to help people organise & participate in the unconference.

Because this should be done before the May event, I'll drop it into that project.

**Comment by vmbrasseur** [comment]
I outlined the general process [in this Google doc](https://docs.google.com/document/d/1P5j9Ozj-5TRh0OzalEDMz-grR1lxCKJXWq1ZvZoJdBA/edit?usp=sharing).

Team Copper will be iterating on it, then I'll turn it into something more readable/accessible.

**Comment by vmbrasseur** [comment]
Rayna says this is A-OK for now.

I'll move this to a markdown file in the workshops repo.

**Comment by vmbrasseur** [comment]
First draft merged. Future work will be in other issues.
